src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -6,19 +6,88 @@
  * За бажанням можна реалізувати інші методи інтрефейсу Map.</p>
  */
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private int realCapacity;
+    private int size;
+    private Node<K, V>[] buckets;
+
+    public MyHashMap() {
+        this.realCapacity = DEFAULT_INITIAL_CAPACITY;
+        this.buckets = new Node[realCapacity];
+    }
 
     @Override
     public void put(K key, V value) {
+        if (size > realCapacity * DEFAULT_LOAD_FACTOR) {
+            resize();
+        }
+        int index = hash(key);
 
+        if (buckets[index] == null) {
```

Comment:
do we need this checking? let's remove it

Reply:
I removed it.